sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/strategy/encrypt/EncryptTable.java

```diff
@@ -153,7 +153,19 @@ public String getCipherColumn(final String logicColumn) {
      * @return optional of sharding encryptor
      */
     public Optional<String> findShardingEncryptor(final String logicColumn) {
-        return columns.containsKey(logicColumn) ? Optional.of(columns.get(logicColumn).getEncryptor()) : Optional.<String>absent();
+        final String originLogicColumnName = getOriginLogicColumnName(logicColumn);
```

Comment:
Why need `final` for `originLogicColumnName` here?

Reply:
Sorry, my problem.